libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/RouteRetrievalEvent.kt

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import android.annotation.SuppressLint
+import android.os.Parcelable
+
+@SuppressLint("ParcelCreator")
```

Comment:
`@SuppressWarnings("ParcelableCreator")` was removed, is it safe?

Reply:
Yes, It is safe